### PR TITLE
JPA HomeWork

### PR DESCRIPTION
**Pull request by DadizhaO**

### CODE REVIEW

File: src/main/java/Lesson17/Order.java [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 
-import javax.persistence.*;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
 import java.io.Serializable;
 import java.math.BigDecimal;
-import java.util.Date;
+import java.time.LocalDate;
```
Comment (SerhiiPasko) [inline src/main/java/Lesson17/Order.java:9]: You crash solution for jdbc. It is not good

File: src/test/java/Lesson17/OfficeDaoImplIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    private Office office1 = new Office(new BigDecimal(120), "city", "region",
+            null, null, new BigDecimal(1223));
+    private Office office2 = new Office(new BigDecimal(120), "city2", "region2",
+            null, new BigDecimal(10101), new BigDecimal(20202));
+
+
+    @Test
+    public void test1InsertOffice() throws SQLException {
+        assertTrue(officeDao.insertOffice(office1));
+        List<Office> offices = manager.createQuery("SELECT o FROM Office o", Office.class).getResultList();
```
Comment (SerhiiPasko) [inline src/test/java/Lesson17/OfficeDaoImplIntegrationTest.java:36]: Bad query - native sql

File: src/test/java/Lesson17/OfficeDaoImplIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        System.out.println(offices);
+        assertTrue(offices.contains(office1));
+        assertEquals(6, offices.size());
```
Comment (SerhiiPasko) [inline src/test/java/Lesson17/OfficeDaoImplIntegrationTest.java:39]: You also depend to dabase. This is checks make logic only for embedded database. In real - never use it (very depend to data in db)

**Review by SerhiiPasko — APPROVED** [review]
Read comments

File: src/main/java/Lesson17/Order.java (hunk 1 above)
Comment (DadizhaO) [reply src/main/java/Lesson17/Order.java:9]: Why crash? For jdbc I have LocalDate too. But on lesson were problems with it and I find a solution for LocalDate.

**Pull request merged**